src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <article class="container">
+      <ul class="calendar calendar--month-length_31 calendar--start-day_sun">
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/235429563-4d06a3f1-833d-4314-8a43-de9ced007c1c.png)
```suggestion
      <ul class="calendar calendar--month-length--31 calendar--start-day--sun">
```